cscs-checks/prgenv/src/alloc_speed.cpp

```diff
@@ -0,0 +1,27 @@
+#include <chrono>
+#include <iostream>
+#include <algorithm>
+
+double test_alloc(size_t n)
+{
+    auto t0 = std::chrono::system_clock::now();
+    /* time to allocate + fill */
+    char* ptr = (char*)std::malloc(n);
```

Comment:
Why not use `new` and wrap it in an `std::unique_ptr`. You'd avoid the `free()` at the end.